news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,167 @@
+
+Both teams start to find their rhythm, winning their respective picks and bringing the match to 6–3 with the United States at match point. The match reaches its climax as Canada picks FM3, where after a stellar team performance from both teams, the United States clinch the win by a mere 23,000 points, securing their crown as the 4WC champions for the second consecutive year and solidifying their status as the powerhouse of the 4-digit tournament scene.
+
+## Interviews
+
+Here are some wise words from a few people involved in various aspects of the tournament.
+
+::{#jimarrah}::
+
+[![](/wiki/shared/news/2024-06-24-4-digit-world-cup-2024-concludes/jimarrah-banner.png)](https://osu.ppy.sh/users/11267857)
+
+> Hi, I'm BOOM MEGA WIN, but you're probably more familiar with me as Jimarrah. I've been playing tournaments since May 2020, spending a majority of my time in the 4 digit scene. Recently I've been taking the game more casually, but 4WC 2024 was a big opportunity for me to compete again.
+
+**BOOM MEGA WIN** (a.k.a. Jimarrah) competed for the title as a first-timer on the Canadian roster and with the help of his team finished in second place.
+
+---
+
+*How did it feel to face off against the United States in Grand Finals, knowing that Canada had lost against them last year already?*
+
+> Before going into Grand Finals, we knew that USA was going to be the team to beat for a spot at the top. The heartbreaking defeat on tiebreaker last year gave us hope that USA was a contestable, and beatable team. Knowing this, we put our best effort into preparing for the weekend. We knew that gimmick was going to be our crutch, and main focus in our picking strategy. One breakpoint could be the single deciding factor in the match, and any mistake could have a grave cost.
+> 
+> In the end, we still lost the hard-fought battle, but USA's paramount performance was absolutely deserving of the win. After the match, we were still in high spirits. We still knew that second place was something to be proud of and that we had an amazing run through the tournament.
+
+*Losing in the Round of 16 against China surely was a massive hit. Have you had any doubts in yourself, your team and how did you manage to bounce back together with your team?*
+
+> This year was an extremely competitive year for 4WC, with a majority of teams and players having major improvements since 4WC 2023. We knew that underestimating the bracket could result in an upset and Canada dropping down to the loser's bracket earlier than expected.
```

Comment:
```suggestion
> This year was an extremely competitive year for 4WC, with a majority of teams and players having major improvements since 4WC 2023. We knew that underestimating the bracket could result in an upset and Canada dropping to the loser's bracket earlier than expected.
```